README.md

```diff
@@ -33,22 +33,40 @@ If you installed Docker for Mac, `docker-compose` is also installed. Otherwise i
 4) [Git](https://git-scm.com/)
 
 
-## Setup
+## Installation
+
+If you're on OSX, the preferred installation method is via homebrew:
+
+```
+$ brew tap artsy/formulas
+$ brew install hokusai
+```
+
+If you've previously installed hokusai via an alternate installation method, you may need to force the `link` step. If you installed hokusai via pip, you may also want to cleanup that installation:
 
-1) Install Hokusai via pip with `pip install hokusai` and `hokusai` will be installed on your `PATH`.
+```
+$ pip uninstall hokusai
+$ brew link --overwrite hokusai
+```
+
+### Alternate Installation Methods
+
+#### Via pip
 
 Note: If installing via pip fails due to pip failing to upgrade your system Python packages, try running `pip install hokusai --ignore-installed`.
 
-Alternatively, you can workaround a pure Python installation and install a binary for OSX by simply downloading it from https://artsy-provisioning-public.s3.amazonaws.com/hokusai/hokusai-{version}-{platform}-x86_64 and adding it to your `$PATH`, i.e. to download the latest version, run:
+#### Via curl
+
+Note: You may need to adjust the target desination to match a directory in your `$PATH`.
 
 ```
 curl --silent https://artsy-provisioning-public.s3.amazonaws.com/hokusai/hokusai-latest-$(uname -s)-$(uname -m) -o /usr/local/bin/hokusai && chmod +x /usr/local/bin/hokusai
 ```
 
+## Setup
 
-2) [Configure your AWS credentials](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#configuring-credentials).
-
-3) Run `hokusai configure --kubectl-version <kubectl version> --s3-bucket <bucket name> --s3-key <file key>`.  You'll need to provide the kubectl version matching your Kubernetes deployments, as well as the S3 bucket name and key of your org's kubectl config file. System administrators: see [Administering Hokusai](./docs/Administering_Hokusai.md) for instructions on preparing AWS, Kubernetes, and publishing a kubectl config file. Artsy devs: see [these artsy/README docs](https://github.com/artsy/README/blob/master/playbooks/hokusai.md) for the current way to install and configure hokusai.
+1. [Configure your AWS credentials](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#configuring-credentials).
```

Comment:
These are both under #1

Reply:
When writing ordered lists in Markdown, the numbers actually aren't rendered directly. Markdown automatically handles the rendering for you. The numbers could even be out of order, or all `3.` and Markdown will still render 1, 2, 3.

This is nice because when updating a list, you don't have to touch items unrelated to your change.

source: https://daringfireball.net/projects/markdown/syntax#list

![screenshot 1560359596](https://user-images.githubusercontent.com/123595/59371844-156f7880-8d14-11e9-9dde-8cc4d3d43a81.png)